### PR TITLE
Api to notify plugin when a dump is upload

### PR DESCRIPTION
**Pull request by ramonberrutti**
The only problem here is that you need sv_hibernate_when_empty 0, Instance when a player enter, will push the function.

This I made to take actions when a server crash, So I know that is crashed, and get notify.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
 /breakpad
+/package
+/test
+/test-crash-dump-generation
+/accelerator.ext
+
+.ambuild
+.vscode
+
+pdblog.txt
```
Comment (asherkin) [inline .gitignore]: FYI, you shouldn't be building in the checkout directory - create a directory for the build (either outside the checkout directory, or `build/` which is already ignored) and run `./configure.py` from there.

File: extension/extension.cpp [hunk 2]
```diff
@@ -394,6 +413,14 @@ class UploadThread: public IThread
 
 			if (uploaded) {
 				count++;
+
+				crashIdPack *pack = new crashIdPack;
+				pack->id = count;
+				memcpy(pack->crashId, response + 10, 14);
```
Comment (asherkin) [inline extension/extension.cpp]: This isn't safe, the message from the server is free-form text and subject to change at any time.

Comment (ramonberrutti) [reply extension/extension.cpp]: yes, I forget only to check the response if less than 14 characters... Also that I can know if you will change the Api, I can add a parse to get the ID with or without the -

**Comment by asherkin** [comment]
> This I made to take actions when a server crash, So I know that is crashed, and get notify.

It would be extremely beneficial to my planning if you could tell me exactly what you do with these notifications.

**Comment by ramonberrutti** [comment]
I manager mora than 1000 servers of CSGO, I build a script that notify to me when a server restart, but now I see that so many servers restart by updating our but anothers things... I only want to recive a notification when a server crash and not when restart, so I can see what is happening... Yes I always refresh my dashboard to see new reports, but I want something in real time so the support of the company can know what is happening. I think that if you add a webhook on your webpage will be excelent.

File: accelerator.inc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * 
+ * @param num           Upload index file.
+ * @param crashId       Id that Web assign to that crash.
+ */
+forward void OnCrashUpdated(int num, const char [] crashId );
+
+/**
+ * Do not edit below this line!
+ */
+public Extension:__ext_accelerator =
```
Comment (Bara) [inline accelerator.inc]:
```suggestion
public Extension __ext_accelerator = 
```

**Comment by Cruze03** [comment]
Hello, why isn't this merged? I really needed that forward
`forward void OnCrashUpdated(int num, const char [] crashId );`

**Pull request closed without merging**